eli5/ipython.py

```diff
@@ -268,14 +291,30 @@ def show_prediction(estimator, doc, **kwargs):
     """
     format_kwargs, explain_kwargs = _split_kwargs(kwargs)
     expl = explain_prediction(estimator, doc, **explain_kwargs)
-    html = format_as_html(expl, **format_kwargs)
-    return HTML(html)
+    if expl.image is not None and expl.heatmap is not None:
+        # dispatch to image display implementation
+        if display_prediction_image is not None:
+            return display_prediction_image(expl, **format_kwargs)
+        else:
+            print('Missing dependencies. Can not display as image.')
```

Comment:
I think it's better to ``warn`` and not print, and also to specify which dependencies are missing.

Reply:
https://github.com/TeamHG-Memex/eli5/pull/315/commits/d2ed75be5e2ecf19b62b20da0dfa2fcc93079c43